src/gql/provider.tsx

```diff
@@ -62,6 +62,49 @@ const Provider: React.FC = ({ children }) => {
           assets: {
             keyArgs: ["teamId", "keyword", "sort", "pagination", ["first", "last"]],
 
+            merge(existing, incoming, { readField }) {
```

Comment:
Here you duplicate the exact same merge function. Please pull it out of the Provider down with the offsetFromCursor function and use it as a function rather than copy/pasting.

Reply:
extracting merge , stopping the merging

Reply:
when I pulled it out it doesn't work the merging

Reply:
I made this change and pushed to the PR. Please look at the changes to see what I meant ;)

Reply:
yes I get that , I was defining the third parameter in different way that is why it didn't work